server/test/unit/server/async/test_app.py

```diff
@@ -1,11 +1,16 @@
 """
 This module contains tests for the pulp.server.async.app module.
 """
-import unittest
-
 import mock
+import unittest
```

Comment:
Since these two imports are part of the standard library, they should be in the first group of imports. ``mock`` should be alone in the second group.